### PR TITLE
New package: brother-brlaser-4

### PR DESCRIPTION
**Pull request by jaredycalves**

### CODE REVIEW

File: srcpkgs/brother-brlaser/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'brother-brlaser'
+pkgname=brother-brlaser
+version=4
+revision=1
+create_wrksrc=yes
+build_wrksrc="brlaser-${version}"
+build_style=cmake
+makedepends="cmake cups-devel"
```
Comment (Johnnynator) [inline srcpkgs/brother-brlaser/template]: cmake does get pulled in by the build_style, just remove it from makedepends.

File: srcpkgs/brother-brlaser/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'brother-brlaser'
+pkgname=brother-brlaser
+version=4
+revision=1
+create_wrksrc=yes
```
Comment (Johnnynator) [inline srcpkgs/brother-brlaser/template]: why create_wrksrc?. `wrksrc="brlaser-${version}"` should do the job.

File: srcpkgs/brother-brlaser/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+build_wrksrc="brlaser-${version}"
+build_style=cmake
+makedepends="cmake cups-devel"
+depends="cups ghostscript avahi avahi-utils"
+short_desc="Brother laser printer driver"
+maintainer="Jaredy <jaredycalves@gmail.com>"
+license="GPL-3.0-or-later"
```
Comment (Johnnynator) [inline srcpkgs/brother-brlaser/template]: `GPL-2.0-or-later`
```
// brlaser is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 2 of the License, or
// (at your option) any later version.
```

File: srcpkgs/brother-brlaser/template (hunk 2 above)
Comment (Johnnynator) [reply srcpkgs/brother-brlaser/template]: Just checked it locally, it does work, you need to change `build_wrksrc` to `wrksrc`

Comment (jaredycalves) [reply srcpkgs/brother-brlaser/template]: Thanks, changed.

File: srcpkgs/brother-brlaser/template [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'brother-brlaser'
+pkgname=brother-brlaser
+version=4
+revision=2
```
Comment (maxice8) [inline srcpkgs/brother-brlaser/template]: revision should be 1

Comment (jaredycalves) [reply srcpkgs/brother-brlaser/template]: Fixed

**Pull request merged**